keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+ * |--------+------+------+------+------+------|  26  |    |  58  |------+------+------+------+------+--------|
+ * |   20   |  21  |  22  |  23  |  24  |  25  |      |    |      |  59  |  60  |  61  |  62  |  63  |   64   |
+ * `--------+------+------+------+------+-------------'    `-------------+------+------+------+------+--------'
+ *   |  27  |  28  |  29  |  30  |  31  |                                |  65  |  66  |  67  |  68  |  69  |
+ *   `----------------------------------'                                `----------------------------------'
+ *                                       ,-------------.  ,-------------.
+ *                                       |  32  |  33  |  |  70  |  71  |
+ *                                ,------+------+------|  |------+------+------.
+ *                                |      |      |  34  |  |  72  |      |      |
+ *                                |  35  |  36  |------|  |------|  74  |  75  |
+ *                                |      |      |  37  |  |  73  |      |      |
+ *                                `--------------------'  `--------------------'
+ */
+{}
+
+}};
+    """.format(",\n\n".join(layer_output),)
+
+def unpack_function(f):
+    name = f.name.name
+    args = []
+    for arg in f.args.exprs:
+        if type(arg) is pycparser.c_ast.Constant:
+            args.append(arg.value)
+        elif type(arg) is pycparser.c_ast.ID:
+            args.append(arg.name)
+    return "{}({})".format(name, ",".join(args))
+
+def transform_raw_key_symbol(raw):
+    if type(raw) is pycparser.c_ast.ID:
+        if raw.name in ["KC_TRNS", "KC_TRANSPARENT"]:
+            return "_______"
+        return raw.name
+    elif type(raw) is pycparser.c_ast.FuncCall:
+        return unpack_function(raw)
+
+def transform_layer(layer):
+    transformed = [transform_raw_key_symbol(k) for k in layer.expr.args.exprs]
+    return transformed
+
+def pretty_output_layer(layer, keys):
+    formatted_key_symbols = """
+    // left hand
+    {},
+    {},
+    {},
+    {},
+     {},
+
+    // left thumb
+    \t{},
+    \t\t{},
+    \t{},
+
+    // right hand
+    {},
+    {},
+    {},
+    {},
+     {},
+
+    // right thumb
+    {},
+    {},
+    {}
+""".format(
+        # left hand
+        ", ".join(keys[0:7]),
+        ", ".join(keys[7:14]),
+        ", ".join(keys[14:20]),
+        ", ".join(keys[20:27]),
+        ", ".join(keys[27:32]),
+        # left thumb
+        ", ".join(keys[32:34]),
+        keys[34],
+        ", ".join(keys[35:38]),
+
+        # right hand
+        ", ".join(keys[38:45]),
+        ", ".join(keys[45:52]),
+        ", ".join(keys[52:58]),
+        ", ".join(keys[58:65]),
+        ", ".join(keys[65:70]),
+        # right thumb
+        ", ".join(keys[70:72]),
+        keys[72],
+        ", ".join(keys[73:76]),
+    )
+
+    s = "[{}] = LAYOUT_ergodox({})".format(layer, formatted_key_symbols)
```

Comment:
```suggestion
    return f"[{layer}] = LAYOUT_ergodox({formatted_key_symbols})"
```
f-string, no need for variable.